modules/renter/contractor/contractor.go

```diff
@@ -56,6 +59,13 @@ func (c *Contractor) Allowance() modules.Allowance {
 	return c.allowance
 }
 
+// Spending returns the number of coins spent on file contracts.
+func (c *Contractor) Spending() (types.Currency, types.Currency) {
```

Comment:
because you are returning two of the same type that mean different things, the return values should definitely be named in the function definition, just for clarity's sake.